storage/integration_test.go

```diff
@@ -3879,15 +3898,6 @@ func TestIntegration_CustomTime(t *testing.T) {
 		if _, err := obj.Update(ctx, ObjectAttrsToUpdate{CustomTime: earlierTime}); err == nil {
 			t.Fatalf("backdating CustomTime: expected error, got none")
 		}
-
-		// Zero value for CustomTime should be ignored. Set TemporaryHold so that
-		// we don't send an empty update request, which is invalid for gRPC.
-		if _, err := obj.Update(ctx, ObjectAttrsToUpdate{TemporaryHold: false}); err != nil {
-			t.Fatalf("empty update: %v", err)
-		}
-		if err := checkCustomTime(laterTime); err != nil {
```

Comment:
I think we should still check that custom time doesn't change after an empty update right?

Reply:
Yeah... I figured we test empty updates elsewhere in several places, but I guess we don't actually set a CustomTime before the empty update, so I've added this back.